lab02/sections/discussion.tex

```diff
@@ -1 +1,14 @@
 \section{Discussion}\label{sec:discussion}
+A phase-lag circuit was built of an identical topology to that in Figure 1.
```

Comment:
You can insert dynamic references by using `\ref{name}` where `name` corresponds to the `\label{name}` field in the figure, section, table, whatever you're referencing.